DelayedClipboard/tag-security-privacy-delayed-clipboard-rendering.md

```diff
@@ -0,0 +1,100 @@
+### Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+For standard formats, this feature doesn't expose any information about the source and destination apps during copy-paste operation. However, for web custom formats, since it's restricted to a specific set of apps, during paste it could expose the source app where the user copied the data from. When a format is delay rendered, the system clipboard requests the data for that format during paste. The source app must provide the data for the delay rendered format when it is requested by the system clipboard, else, the paste would fail in the destination app.
+For more discussion, please see https://github.com/w3c/editing/issues/439.
+
+To use delay rendering via the async clipboard API, all the existing restrictions in async clipboard APIs related to secure context, permissions etc apply.
+
+### 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes.
+
+### 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
```

Comment:
Is this true for delay rendering of web custom formats?

Reply:
I wasn't sure if a trusted app (where the user is performing the paste) knowing where the user copied the data from would fall under PII. I'm not sure how to describe that part so I mentioned it as No.

Reply:
Done.